src/main/java/com/spikes2212/command/genericsubsystem/MotoredGenericSubsystem.java

```diff
@@ -15,48 +15,80 @@
 public class MotoredGenericSubsystem extends GenericSubsystem {
 
     protected MotorControllerGroup motorControllerGroup;
-    protected RootNamespace rootNamespace;
 
     /**
-     * Constructs a new instance of MotoredGenericSubsystem with the given {@code RootNamespace}'s name, the given
-     * maxSpeed supplier, the given minSpeed supplier and the given {@code MotorController}s.
+     * Constructs a new instance of {@link MotoredGenericSubsystem} with the given {@link RootNamespace}'s name, the given
+     * minSpeed supplier, the given maxSpeed supplier and the given {@link MotorController}s.
```

Comment:
"with the given minimum and maximum speed suppliers"
do it for everywhere else as well